apps/navstar_l4lb/src/navstar_l4lb.erl

```diff
@@ -1,4 +1,4 @@
--module(minuteman_sup).
+-module(navstar_l4lb).
```

Comment:
maybe call it navstar_l4lb_sup to follow the convention (and rename the file)? or is the convention different in navstar than it was in minuteman?